src/doc/yaml/enrollment.yaml

```diff
@@ -0,0 +1,666 @@
+        compression:
+          $ref: '#/components/schemas/CompressionType'
+    DocumentPart:
+      type: object
+      properties:
+        pages:
+          type: array
+          description: The pages included in this part. Can be a single page number, or a list.
+          minItems: 1
+          items:
+            type: integer
+        data:
+          type: string
+          format: byte
+          description: Base64-encoded data of the document
+        dataRef:
+          type: string
+          format: uri
+          description: URI to the data
+          example: "http://server.com/buffer?id=00003"
+        width:
+          type: integer
+          description: the width of the image in pixels
+        height:
+          type: integer
+          description: the height of the image in pixels
+        captureDate:
+          type: string
+          format: date
+          example: "2019-05-21"
+        captureDevice:
+          type: string
+          description: A string identifying the device used to capture the document part
+        mimetype:
+          type: string
+          description: the nature and format of a document. The mime type definitions should be in compliance with RFC 6838.
+    DocumentData:
+      type: object
+      required:
+        - documentType
+        - parts
+      properties:
+        documentType:
+          $ref: '#/components/schemas/DocumentType'
+        instance:
+          type: string
+          description: "Used to separate two distincts documents of the same type (ex: two passports)"
+        parts:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/DocumentPart'
+    Expression:
+      type: object
+      required:
+        - attributeName
+        - operator
+        - value
+      properties:
+        attributeName:
+          type: string
+        operator:
+          type: string
+          enum: ['<', '>', '=', '>=', '<=']
+        value:
+          oneOf:
+          - type: string
+          - type: integer
+          - type: number
+          - type: boolean
+    Expressions:
+      type: array
+      items:
+        $ref: '#/components/schemas/Expression'
+    CompressionType:
+      type: string
+      enum: [NONE, WSQ, JPEG, JPEG2000, PNG]
+    ImpressionType:
+      type: string
+      enum:
+        - LIVE_SCAN_PLAIN
+        - LIVE_SCAN_ROLLED
+        - NONLIVE_SCAN_PLAIN
+        - NONLIVE_SCAN_ROLLED
+        - LATENT_IMPRESSION
+        - LATENT_TRACING
+        - LATENT_PHOTO
+        - LATENT_LIFT
+        - LIVE_SCAN_SWIPE
+        - LIVE_SCAN_VERTICAL_ROLL
+        - LIVE_SCAN_PALM
+        - NONLIVE_SCAN_PALM
+        - LATENT_PALM_IMPRESSION
+        - LATENT_PALM_TRACING
+        - LATENT_PALM_PHOTO
+        - LATENT_PALM_LIFT
+        - LIVE_SCAN_OPTICAL_CONTACTLESS_PLAIN
+        - OTHER
+        - UNKNOWN
+    DocumentType:
+      type: string
+      enum:
+        - ID_CARD
+        - Passport
```

Comment:
Could you sync with the PR interface which has:
```
      type: string
      description: Type of document. Can be extended.
      enum: [ID_CARD, PASSPORT]
```
What is important is to indicate this list can be extended.